src/components/Weapons.js

```diff
@@ -68,8 +95,76 @@ const Weapons = ({
           <IconButton onClick={addWeapon}>
             <i className="material-icons">done</i>
           </IconButton>
+           <Row>
+           <h2>Can&apos;t find your weapon?</h2>
+           <Button variant="contained" color="primary" onClick={() => setAddingUnique(true)}>Add Unique Weapon</Button>
+           </Row>
            </Row>
       }
+      {
+        isAddingUnique
+        && <>
+        <Row>
+          <h3>Add Unique Weapon</h3>
+          <Spacer />
+        </Row>
+          <Row>
+          <BasicBox style={{ margin: '.75rem' }}>
```

Comment:
Basic box restricts the size, we may not want this for every field. Generally basic box is good for small/numerical values.